### ISSUE TITLE
Unnecessary Snake Case?

### ISSUE THREAD

**Issue by intrepidws** [bezhanSalleh/filament-shield#70]
In the `ViewShieldSettings.php` file, the following code exists:

```php
Forms\Components\TextInput::make('super_admin.name')
    ->label(__('filament-shield::filament-shield.labels.super_admin.text_input'))
    ->afterStateHydrated(function ($set, $state) {
        $set('super_admin.name', Str::of($state)->snake()->toString());
    })
    ->visible(fn ($get) => $get('super_admin.enabled'))
    ->required(fn ($get) => $get('super_admin.enabled')),
```

The `afterStateHydrated()` portion of that code does not exist for this similar block of code:

```php
Forms\Components\TextInput::make('filament_user.name')
    ->label(__('filament-shield::filament-shield.labels.filament_user.text_input'))
    ->visible(fn ($get) => $get('filament_user.enabled'))
    ->required(fn ($get) => $get('filament_user.enabled'))
```

It doesn't seem to make that it's necessary to convert `super_admin.name` to snake case. Regardless, it would seem that either both `super_admin.name` and `filament_user.name` should be converted to snake case or both should not be. Happy to submit a PR if you can tell me which you'd prefer (and if there is a reason to convert to snake case or not).

For what it's worth, I think it would be nice if the `name` variables did not need to be snake case, as I prefer naming something `Super Admin` versus `super_admin`- especially when that is what users will see when assigning roles via the admin.

**Comment by intrepidws** [comment]
After looking around a bit more, it seems that the conventions surrounding the Role `name` field are pretty inconsistent. When creating a new role, `name` is made lowercase:

```php
Forms\Components\TextInput::make('name')
    ->label(__('filament-shield::filament-shield.field.name'))
    ->required()
    ->maxLength(255)
    ->afterStateUpdated(fn (Closure $set, $state): string => $set('name', Str::lower($state))),
```

And when displayed in the table, it's shown as headline case:

```php
Tables\Columns\BadgeColumn::make('name')
    ->label(__('filament-shield::filament-shield.column.name'))
    ->formatStateUsing(fn ($state): string => Str::headline($state))
    ->colors(['primary'])
    ->searchable(),
```

I could be missing it entirely, but it doesn't seem like there's a reason to enforce _ANY_ case at all with the name of a role?

Again, I'm happy to PR the changes but would be interested in hearing how you'd prefer to handle this situation before I do.

**Comment by bezhanSalleh** [comment]
these have been there from the start, didn't even notice them until v2.
and has been refactored in #69 .. which will be available in the next release 2.0.5 or 2.0.6 😇
